### ISSUE TITLE
Gradle wrapper should be executable

### ISSUE THREAD

**Issue by uzilan** [exercism/exercism#4991]
When downloading exercises in the kotlin track (and maybe java as well?), which contain gradle wrapper files (gradlew and gradlew.bat) - those files are non-executable and must be chmoded before mentees can run them. 

This was discusses in  https://github.com/exercism/cli/issues/276 before. 

To me, this doesn't make much sense. Either include an executable gradle wrapper or avoid including it altogether, but giving the mentees a wrapper that they need to chmod before being able to run it means we need to instruct them on how to do so and makes solving solutions more complicated than it needs to be.

**Comment by sshine** [comment]
@glennj: We are affected by this on the Tcl track and should probably revise testing hints to use

    tclsh <exercise>.test

until this improvement is made.

**Comment by dector** [comment]
@uzilan Is it still an issue?

**Comment by uzilan** [comment]
Yeah I think so. Try redownloading the acronym exercise for example:
```
ll acronym                                         
total 40
-rw-r--r--  1 uzilan  staff   922B Nov 16 23:18 README.md
-rw-r--r--  1 uzilan  staff   531B Nov 16 23:18 build.gradle
drwxr-xr-x  3 uzilan  staff    96B Nov 16 23:18 gradle
-rw-r--r--  1 uzilan  staff   5.2K Nov 16 23:18 gradlew
-rw-r--r--  1 uzilan  staff   2.1K Nov 16 23:18 gradlew.bat
drwxr-xr-x  4 uzilan  staff   128B Nov 16 23:18 src
```
So this won't work obviously:
```
 ./gradlew clean test
zsh: permission denied: ./gradlew
```

**Comment by dector** [comment]
Thanks. I'll investigate this.

**Comment by dector** [comment]
Is it possible to transfer this issue to Kotlin repo so we can track all Kotlin-related issues in one place?

**Comment by dector** [comment]
@uzilan Created follow-up issue for [cli](https://github.com/exercism/cli/issues/903).
